spec/features/map/nlp_spatial_temporal_extraction_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+describe "Spatial and temporal extraction", reset: false do
```

Comment:
This is a good start. Do I remember talk of being able to pull both spatial and temporal out of the same search? If so be sure to add that test as well